clang/tools/dpct/DpctOptRules/cmake_script_migration_rule.yaml

```diff
@@ -2814,3 +2814,34 @@
       MatchMode: Full
       In: NVIDIA
       Out: IntelLLVM
+
+- Rule: rule_CUDA_PROPAGATE_HOST_FLAGS
+  Kind: CMakeRule
+  Priority: Fallback
+  CmakeSyntax: CUDA_PROPAGATE_HOST_FLAGS
+  In: ${func_name}(${value})
+  Out: ${func_name}(${value})
+  Subrules:
+    value:
+      MatchMode: Full
+      In: CUDA_PROPAGATE_HOST_FLAGS
+      Out: DUMP_VAR
+
+- Rule: rule_CUDA_USE_STATIC_CUDA_RUNTIME
+  Kind: CMakeRule
+  Priority: Fallback
+  CmakeSyntax: CUDA_USE_STATIC_CUDA_RUNTIME
```

Comment:
CUDA_USE_STATIC_CUDA_RUNTIME should be removed.

Reply:
![image](https://github.com/user-attachments/assets/ee6c238c-38af-4d51-b055-1df1e789e72f)

Removing CUDA_USE_STATIC_CUDA_RUNTIME Entire might cause a CMake config error. I suggest we keep a dummy variable since it is not applicable to SYCL

Reply:
Done